components/lib/crash/src/main/java/mozilla/components/lib/crash/notification/CrashNotification.kt

```diff
@@ -28,42 +31,36 @@ internal class CrashNotification(
 ) {
     fun show() {
         val pendingIntent = PendingIntent.getActivity(
-            context, 0, CrashPrompt.createIntent(context, crash), 0
+                context, PROMPT_REQUEST_CODE, CrashPrompt.createIntent(context, crash), 0
         )
 
-        val channel = ensureChannelExists()
+        val reportPendingIntent = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            PendingIntent.getForegroundService(
+                    context, REPORT_REQUEST_CODE, CrashPrompt.createReportIntent(context, crash), 0
+            )
+        } else {
+            PendingIntent.getService(
+                    context, REPORT_REQUEST_CODE, CrashPrompt.createReportIntent(context, crash), 0
+            )
+        }
```

Comment:
Are the request codes needed? We never seem to not read them in this setup.

Reply:
Agreed.  I'm launching a different receiver with the PendingIntent so I don't need to have different request code anymore.